shynet/analytics/views/ingress.py

```diff
@@ -129,6 +129,9 @@ def get(self, *args, **kwargs):
             )
         )
         heartbeat_frequency = settings.SCRIPT_HEARTBEAT_FREQUENCY
+        dnt = self.request.META.get("HTTP_DNT", "0").strip() == "1"
```

Comment:
@doamatto do we need to also handle the GPC header here? You're the expert on this :)

Reply:
Is GPC header really about tracking? It looks like it's about selling and sharing data. According to MDN:
`The Sec-GPC (Global Privacy Control) request header indicates whether the user consents to a website or service selling or sharing their personal information with third parties.`

Reply:
True. In that case we should consider not lumping it in with DNT on the server side either.

Reply:
:+1:

Reply:
Global Privacy Control is an implementation of the CCPA [1] and GDPR [2] meant to be the equivalent of "Do Not Sell" and "Object to Processing."[3] It's seen as an actual standard for the DNT header, which, even though we call it "Do No Track", doesn't really hold any merit outside of that people *should* respect it, not what they *have* to respect it.

GPC can be seen as a "Do Not Track" header— as far as I know, it's meant to even replace it, considering the amount of legistlation behind it.

[1]: GPC signal conveys limitation of sale for the browser, device, or consumer; I have zero clue what section exactly, but I imagine that's on the GPC's website since it's been basically endorsed by the attorney general and other CA figures.
[2]: GPC signal conveys limitation of sale or sharing of user's personal data; Art. 7 & 21
[3]: https://globalprivacycontrol.org/

cc @milesmcc

Reply:
Maybe we should treat GPC as DNT for sake of simplicity. There is always an option to ignore DNT. 
Another option is to add new setting to enable GPC as DNT. 
Ideally Shynet should not give possibility to track user personal data.